packages/synapse-react-client/src/mocks/msw/handlers/challengeHandlers.ts

```diff
@@ -12,11 +12,16 @@ import {
   mockRegisteredChallengeTeams,
 } from '../../challenge/mockChallenge'
 import { MOCK_CHALLENGE_PARTICIPANT_TEAM_ID } from '../../team/mockTeam'
-import { uniqueId } from 'lodash-es'
+import BasicMockedCrudService from '../util/BasicMockedCrudService'
```

Comment:
Created a new utility class `BasicMockedCrudService` to implement in-memory database logic, and use it where it's easy to replace existing code.